packages/material-ui/src/unstable_useWidth/unstable_useWidth.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import debounce from 'debounce'; // < 1kb payload overhead when lodash/debounce is > 3kb.
+import { ThemeContext } from '@material-ui/styles/ThemeProvider';
```

Comment:
Should I add it as a peer dependency?

Reply:
I would say as a dependency.

Reply:
The reason I suggested a peer dependency is because the only thing that directly depends on it is useWidth.

Where do you suggest adding that it is now a dependency?

Reply:
The issue with peer dependencies is that it displays a warning for everybody when missing. In this case, we can ignore the issue. We can just add a warning about it in the documentation. We will add the dependency/peer dependency later on.